opengrok-web/src/main/java/org/opengrok/web/api/v1/filter/IncomingFilter.java

```diff
@@ -86,14 +97,38 @@ public void filter(final ContainerRequestContext context) {
         if (request == null) { // happens in tests
             return;
         }
+
         String path = context.getUriInfo().getPath();
+
+        if (request.isSecure()) {
+            String authHeaderValue;
+            if ((authHeaderValue = request.getHeader(HttpHeaders.AUTHORIZATION)) != null &&
```

Comment:
I've noticed this construct a few times in OpenGrok and I think it makes things a bit unreadable. Why not rewrite it as:
```java
String authHeaderValue = request.getHeader(HttpHeaders.AUTHORIZATION);
if (authHeaderValue != null && authHeaderValue.startsWith(BEARER)) {
   ...
```
Then the definition is directly tied to declaration.

Reply:
yep, that looks nicer.